airbyte-config/config-models/src/main/resources/types/SyncStats.yaml

```diff
@@ -21,3 +21,7 @@ properties:
     type: integer
   recordsCommitted:
     type: integer # if unset, committed records could not be computed
+  meanSecondsBeforeSourceStateMessageEmitted:
+    type: integer
```

Comment:
Tracking an average with an integer looks a bit unusual.
Any concerns about precision loss (especially since we also track the rolling average in an int)?

Reply:
The PRD does specify seconds, and I do think seconds are enough for these purposes since we're expecting to see a kind of high # of seconds for these means & maxes

Reply:
Makes sense, I think I missed the typing section of the PRD.